backend/vendor/modules.txt

```diff
@@ -72,7 +72,7 @@ github.com/go-chi/cors
 # github.com/go-chi/render v1.0.1
 ## explicit
 github.com/go-chi/render
-# github.com/go-pkgz/auth v0.10.2
+# github.com/go-pkgz/auth v0.10.3-0.20200615230206-4150d016cbbf
```

Comment:
`0.11.0` should have what you are looking for.

Reply:
Actually, it's already in this repo so the update is no longer necessary.

Reply:
@paskal Reverted to the version in master